fe/fe-core/src/main/java/com/starrocks/backup/BackupJob.java

```diff
@@ -509,12 +516,16 @@ private void prepareAndSendSnapshotTask() {
             List<Table> copiedTables = Lists.newArrayList();
             for (TableRef tableRef : tableRefs) {
                 String tblName = tableRef.getName().getTbl();
-                OlapTable tbl = (OlapTable) globalStateMgr.getLocalMetastore()
-                            .getTable(db.getFullName(), tblName);
+                Table tbl = globalStateMgr.getLocalMetastore().getTable(db.getFullName(), tblName);
+                if (tbl.isOlapView()) {
+                    copiedTables.add(tbl);
```

Comment:
not copied，is there consistent problem ？

Reply:
Updated, deep copy is prefer